TeachingRecordSystem/src/TeachingRecordSystem.SupportUi/Pages/Persons/PersonDetail/Timeline/Events/EventWithUser.cs

```diff
@@ -0,0 +1,23 @@
+namespace TeachingRecordSystem.SupportUi.Pages.Persons.PersonDetail.Timeline.Events;
+
+/// <summary>
+/// Flattened out record to allow Event, TRS User and DQT User to be returned in a single SQL query
+/// </summary>
+public record EventWithUser
+{
+    public required Guid EventId { get; init; }
+    public required string EventName { get; init; }
+    public required DateTime EventCreated { get; init; }
+    public required string EventPayload { get; init; }
+    public required bool EventPublished { get; set; }
```

Comment:
Do we need `EventId`, `EventCreated` or `EventPublished`?

Reply:
Not how it's used currently we don't

Reply:
Removed